2022/06/14 18:16:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/index.php?r=site/users
HTTP_REFERER=http://localhost/demoyii/index.php
---
2022/06/14 18:22:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/index.php?r=site/users
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:25:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/site/users
HTTP_REFERER=http://localhost/demoyii/index.php/site/index
---
2022/06/14 18:26:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/site/users
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:29:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/site/users
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:29:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:29:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:29:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:29:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:30:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:30:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:30:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:30:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:30:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:30:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:30:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:30:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/site/page?view=about
---
2022/06/14 18:30:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:30:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "assets/5e2914df/jquery.js".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('assets/5e2914df...')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/assets/5e2914df/jquery.js
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:30:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:30:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "assets/5e2914df/jquery.yiiactiveform.js".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('assets/5e2914df...')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/assets/5e2914df/jquery.yiiactiveform.js
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:30:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:30:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:30:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/site/users
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:30:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:30:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:30:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:30:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:30:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:30:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:30:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:30:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:31:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/site/users
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:31:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:31:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:31:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:31:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:31:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/site/users
HTTP_REFERER=http://localhost/demoyii/site/contact
---
2022/06/14 18:31:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:31:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:31:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:31:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/site/users
---
2022/06/14 18:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/index.php/site/index
---
2022/06/14 18:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/index.php/site/index
---
2022/06/14 18:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/index.php/site/index
---
2022/06/14 18:32:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/index.php/site/index
---
2022/06/14 18:32:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:32:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:32:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:32:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:32:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/main.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/main.css
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:32:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/screen.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/screen.css
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:32:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/print.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/print.css
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:32:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('css/form.css')
#1 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/demoyii/css/form.css
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:36:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/index.php?r=site/users
HTTP_REFERER=http://localhost/demoyii/
---
2022/06/14 18:47:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "users".' in C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(270): CController->missingAction('users')
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#2 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demoyii\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demoyii/index.php?r=site/users
HTTP_REFERER=http://localhost/demoyii/index.php?r=site/index
---
2022/06/14 22:22:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii-1.1.16\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1474): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(117): CActiveRecord->findAll()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:30:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii-1.1.16\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1474): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(117): CActiveRecord->findAll()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:30:33 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii-1.1.16\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1474): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(117): CActiveRecord->findAll()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:30:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii-1.1.16\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1474): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(117): CActiveRecord->findAll()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:31:07 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000]: Syntax error or access violation: 1115 Unknown character set: 'utf-8'' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii-1.1.16\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1474): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(117): CActiveRecord->findAll()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:31:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000]: Syntax error or access violation: 1115 Unknown character set: 'utf-8'' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii-1.1.16\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1474): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(117): CActiveRecord->findAll()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:31:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000]: Syntax error or access violation: 1115 Unknown character set: 'utf-8'' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii-1.1.16\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1474): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(117): CActiveRecord->findAll()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:31:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000]: Syntax error or access violation: 1115 Unknown character set: 'utf-8'' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii-1.1.16\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(458): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(637): CApplication->getDb()
#5 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(670): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1474): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(117): CActiveRecord->findAll()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:35:05 [error] [system.db.CDbCommand] CDbCommand::fetchColumn() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '*' at line 1. The SQL statement executed was: SELECT COUNT(*) FROM `tbl_user_yii_doc` `t` WHERE *.
in C:\xampp\htdocs\estudosHostweb\protected\views\site\users.php (1)
in C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\estudosHostweb\index.php (13)
2022/06/14 22:35:05 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '*' at line 1. The SQL statement executed was: SELECT COUNT(*) FROM `tbl_user_yii_doc` `t` WHERE *' in C:\xampp\htdocs\yii-1.1.16\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\db\CDbCommand.php(433): CDbCommand->queryInternal('fetchColumn', 0, Array)
#1 C:\xampp\htdocs\yii-1.1.16\framework\db\ar\CActiveRecord.php(1606): CDbCommand->queryScalar()
#2 C:\xampp\htdocs\estudosHostweb\protected\views\site\users.php(1): CActiveRecord->count('*')
#3 C:\xampp\htdocs\yii-1.1.16\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\yii-1.1.16\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(782): CController->renderPartial('users', Array, true)
#7 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(118): CController->render('users', Array)
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#9 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): CController->run('users')
#13 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController('site/users')
#14 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\estudosHostweb\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/estudosHostweb/index.php?r=site/users
HTTP_REFERER=http://localhost/estudosHostweb/index.php
---
2022/06/14 22:39:08 [error] [php] Object of class User could not be converted to string (C:\xampp\htdocs\estudosHostweb\protected\views\site\users.php:1)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(118): SiteController->render()
#3 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionUsers()
#4 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\estudosHostweb\index.php(13): CWebApplication->run()
REQUEST_URI=/estudosHostweb/index.php?r=site/users
in C:\xampp\htdocs\estudosHostweb\protected\views\site\users.php (1)
in C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\estudosHostweb\index.php (13)
2022/06/15 00:09:25 [error] [php] Undefined variable: model (C:\xampp\htdocs\estudosHostweb\themes\classic\views\site\index.php:43)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(32): SiteController->render()
#3 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\estudosHostweb\index.php(13): CWebApplication->run()
REQUEST_URI=/estudosHostweb/index.php?r=site/index
in C:\xampp\htdocs\estudosHostweb\themes\classic\views\site\index.php (43)
in C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php (32)
in C:\xampp\htdocs\estudosHostweb\index.php (13)
2022/06/15 00:09:32 [error] [php] Undefined variable: model (C:\xampp\htdocs\estudosHostweb\themes\classic\views\site\index.php:43)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(32): SiteController->render()
#3 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\estudosHostweb\index.php(13): CWebApplication->run()
REQUEST_URI=/estudosHostweb/index.php?r=site/index
in C:\xampp\htdocs\estudosHostweb\themes\classic\views\site\index.php (43)
in C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php (32)
in C:\xampp\htdocs\estudosHostweb\index.php (13)
2022/06/15 00:09:41 [error] [php] Undefined variable: model (C:\xampp\htdocs\estudosHostweb\themes\classic\views\site\index.php:43)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(32): SiteController->render()
#3 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\estudosHostweb\index.php(13): CWebApplication->run()
REQUEST_URI=/estudosHostweb/index.php?r=site/index
in C:\xampp\htdocs\estudosHostweb\themes\classic\views\site\index.php (43)
in C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php (32)
in C:\xampp\htdocs\estudosHostweb\index.php (13)
2022/06/15 00:09:45 [error] [php] Undefined variable: model (C:\xampp\htdocs\estudosHostweb\themes\classic\views\site\index.php:43)
Stack trace:
#0 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php(32): SiteController->render()
#3 C:\xampp\htdocs\yii-1.1.16\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii-1.1.16\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii-1.1.16\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii-1.1.16\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\estudosHostweb\index.php(13): CWebApplication->run()
REQUEST_URI=/estudosHostweb/index.php?r=site/index
in C:\xampp\htdocs\estudosHostweb\themes\classic\views\site\index.php (43)
in C:\xampp\htdocs\estudosHostweb\protected\controllers\SiteController.php (32)
in C:\xampp\htdocs\estudosHostweb\index.php (13)
